src/psp-migration.md

```diff
@@ -0,0 +1,375 @@
+# How to migrate from Kubernetes PSPs to Kubewarden policies?
+Kubewarden can be used to replace all deprecated [PodSecurityPolicy](https://kubernetes.io/docs/concepts/security/pod-security-policy/). Different from the PSPs, Kubewarden has separated
+policies to achieve the same goal of different fields from the PSPs. The policies to cover each of the PSP configuration fields can be found below in the [mapping table](#mapping-kuberwarden-policies-to-psp-fields).
+
+To start deploying the Kubewarden policies to replace the PSPs, you need to install Kubewarden in your cluster. For that, please, refer to
```

Comment:
Nit: 

```suggestion
To start deploying the Kubewarden policies to replace the PSPs, you need to install Kubewarden in your cluster. For that, refer to
```

(https://developers.google.com/style/tone#politeness-and-use-of-please)